containers/python-3-device-simulator-express/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# Update the VARIANT arg in devcontainer.json to pick a Python version: 3.7, 3.8 or higher
+ARG VARIANT=3.8
+FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
+
+# [Optional] If your requirements rarely change, uncomment this section to add them to the image.
+#
+# COPY requirements.txt /tmp/pip-tmp/
+# RUN pip3 --disable-pip-version-check --no-cache-dir install -r /tmp/pip-tmp/requirements.txt \
+#    && rm -rf /tmp/pip-tmp
+
+# [Optional] Uncomment this section to install additional packages.
+#
+# RUN apt-get update \
+#     && export DEBIAN_FRONTEND=noninteractive \
+#    && apt-get -y install --no-install-recommends <your-package-list-here>
+
+# Create venv for device simulator
+RUN python3 -m venv /opt/vscode/extensions/ms-python.devicesimulatorexpress/venv
```

Comment:
After running this you'll probably want to make sure that the non-root user can use it w/o `sudo`.  e.g.

`chown -R vscode:root /opt/vscode/extensions/ms-python.devicesimulatorexpress`